docs/docs/workflows/steps/triggers/README.md

```diff
@@ -288,12 +288,12 @@ When you're processing HTTP requests, you often don't need to issue any special
 
 #### Customizing the HTTP response
 
-If you need to issue a custom HTTP response from a workflow, **you can use the `$respond()` function in a Code or Action step**.
+If you need to issue a custom HTTP response from a workflow, **you can use the `$.respond()` function in a Code or Action step**.
 
-`$respond()` takes a single argument: an object with properties that specify the body, headers, and HTTP status code you'd like to respond with:
+`$.respond()` takes a single argument: an object with properties that specify the body, headers, and HTTP status code you'd like to respond with:
 
 ```javascript
-$respond({
+$.respond({
```

Comment:
You'll also need to modify this example to use `defineComponent` + the `run` method.

I would also call out here that returning HTTP responses are only available in the Node.js runtime.